importScripts/sampleRecords/photographs.browse.bundles.json

```diff
@@ -6,6 +6,7 @@
         "ca_objects.SpecificLocation.BoxType": {},
         "ca_objects.SpecificLocation.Copied": {
             "convertCodesToDisplayText" : true
-        }
+        },
+        "ca_objects.DateOfCreation": {}
```

Comment:
This file gets passed to the CA webservice asking for these specific fields. This is how the `photographs.destination.json` is created.